purpleair-aqi.js

```diff
@@ -225,27 +303,42 @@ function getAQITrend({ v1: partLive, v3: partTime }) {
   return "arrow.left.and.right";
 }
 
+/**
+ * Constructs an SFSymbol from the given symbolName
+ *
+ * @param {string} symbolName
+ * @returns {object} SFSymbol
+ */
+function createSymbol(symbolName) {
+  const symbol = SFSymbol.named(symbolName);
+  symbol.applyFont(Font.systemFont(20));
+  return symbol;
+}
+
 async function run() {
   const listWidget = new ListWidget();
   listWidget.setPadding(10, 15, 10, 10);
 
   try {
-    console.log(`Using sensor ID: ${SENSOR_ID}`);
+    /** @type {LatLon} */
+    const deviceLocation = await Location.current();
+    const sensorId = await getSensorId(deviceLocation);
+    console.log(`Using sensor ID: ${sensorId}`);
 
-    const data = await getSensorData(API_URL, SENSOR_ID);
+    const data = await getSensorData(sensorId);
     const stats = JSON.parse(data.val);
-    console.log(stats);
+    console.log({ stats });
```

Comment:
By wrapping the variable in a pair of `{}`, it prints the variable's name as well as it's value in the console.

Reply:
Expanding on that explanation, it's equivalent to `{ stats: stats }`. See https://eslint.org/docs/rules/object-shorthand.